p/protoc/ubuntu-20.04/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:20.04
```

Comment:
Can we instead make use of `adoptopenjdk`
Or is this a requirement from upstream/ downstream to be Ubuntu?

Reply:
Apache HBase and Bigtop don't use adoptopenjdk.